markdown/articles/2021/08/22/Non-blue-fair-decks.md

```diff
@@ -0,0 +1,429 @@
+
+#### Post Doomsday 
+
+Like against most green Decks {{Endurance}} is the only real threat to your easy win.  
+That's why it's super important to build safe piles. As long as you resolved Doomsday early, there is no reason to get fancy.  
+Fetches are strong against {{Wasteland}} and {{Collector Ouphe}}, so try to stay away from petals as your mana sources for {{Thassa's Oracle}}.  
+
+## Lands
+
+### Gameplan
+
+Playing the long game is not a winning line.
+
+### Cards to keep in mind
+
+#### Maindeck
+
+<row variant="hand"> {{!Ghost Quarter}} {{!Dark Depths}} {{!Crop Rotation}} {{!Wasteland}} {{!Rishadan Port}} {{!Urza's Saga}}</row>
+
+#### Sideboard
+
+<row variant="hand"> {{!Pyroblast}} {{!Sphere of Resistance}} {{!Choke}} {{!Endurance}} </row>
+
+### Gameplay
+
+#### Pre Doomsday
+
+Lands has super strong mana denial, even more in Game 2 and 3, and will propably always win the long game, but it can switch gears super fast thanks to {{Dark Depths}}.  
+Based on that analysis a fast Doomsday is the way to go. But when you are mulliganing keep in mind, how strong countermagic is against their post sideboard hate pieces.
+
+#### Post Doomsday 
+
+You have 3 main objectives post Doomsday. Have enough mana to cast {{Thassa's Oracle}}. Beat {{Pyroblast}}. Beat {{Endurance}}. Building piles against Lands isn't always easy, and sometimes you have to take some risks, but if you play to your objectives in the priority they are listes above, you should be on a good way to win.  
+You have to learn assess your opponents posibilities, that's not something you'll learn reading this guide. You need to play matches over matches to refine this skill. But i can give you list of usefull tools you propably have in your deck.  
+Fetches are great to play around mana denial. You can "hide" you mana producing lands behind them while thinning your deck.  
+Free Countermagic is your best tool against every form hate spells. From {{Endurance}} over {{Choke}} to {{Sphere of Resistance}}.  
+Artifact mana is super good when you want to end the game fast and you know they don't have ways to mess with it.  
+{{Cavern of Souls}} is an easy counter to Pyroblast.  
+Bounce Spells are the way to go to beat a fast Marit Lage.  
+
+## Mono-Green Cloudpost
+
+### Gameplan
+
+Just slam, and beat Endurance post Doomsday
+
+### Cards to keep in mind
+
+#### Maindeck
+
+<row variant="hand"> {{!Endurance}} {{!Dark Depths}}</row>
+
+#### Sideboard
+
+<row variant="hand"> {{!Mindbreak Trap}}</row>
+
+### Gameplay
+
+#### Pre Doomsday
+
+This will be a short one. Just cast Doomsday asap and play around possible {{Mindbreak Traps}} in sideboarded games. Like in any Dark Depths deck, their plays to creat a 20/20 are super telegraphed.
+
+#### Post Doomsday 
+
+They only have one card to interact with you, {{Endurance}}. You should always have enough time for a pile that beats Endurance and even plays around Mindbreak Trap.
+
+## GW Depths
```

Comment:
`This answers my earlier question. In which case I am not sure that Depths out of reg mav is all that common?